R/alt_aggregate_counts.R

```diff
@@ -0,0 +1,137 @@
+#' Alternate Aggregate UCLA data for website groupings
+#'
+#' Reads the UCLA aggregates counts for states for most recent
+#' data within a given window and reports either state level data or national
+#' data. States include values for the 50 states broken down by carceral type,
+#' prison, ICE, Federal, Juvenile, Psychiatric, and county.
+#'
+#' @param window integer, the day range of acceptable data to pull from, ignored
+#' if all dates is true
+#' @param all_dates logical, get time series data rather than just latest counts
+#' @param week_grouping logical, use weekly grouping for past data? else monthly
+#' @param collapse_vaccine logical, combine vaccine variables for more
+#' intuitive comparisons
+#'
+#' @return data frame with aggregated counts at state or national level
+#'
+#' @examples
+#' \dontrun{
+#' alt_aggregate_counts()
+#' }
+#' @export
+
+alt_aggregate_counts <- function(
+    window = 31, all_dates = FALSE, week_grouping = TRUE,
+    collapse_vaccine = TRUE){
+
+    round_ <- ifelse(week_grouping, "week", "month")
+
+    ucla_df <- read_scrape_data(
+        window = window, all_dates = all_dates, wide_data = FALSE) %>%
+        mutate(Web.Group = case_when(
+            Jurisdiction == "immigration" ~ "ICE",
+            Jurisdiction == "federal" ~ "Federal",
+            Age == "Juvenile" ~ "Juvenile",
+            Jurisdiction == "state" ~ "Prison",
+            Jurisdiction == "psychiatric" ~ "Psychiatric",
+            Jurisdiction == "county" ~ "County",
+            TRUE ~ NA_character_
+        ))
+
+    fac_long_df <- ucla_df %>%
+        filter(State != "Not Available") %>%
```

Comment:
Thoughts on leaving these in and just keeping the state as `Not Available`? Means we'd keep both federal / ice facilities that we haven't gotten around to adding yet, but also federal aggregations (e.g. RRCs) where they're cross-state so the actual assigned state is `Not Available`.

Reply:
I like this idea but I'm not sure, practically, whether this matters. If state = "Not Available", would the numbers show up anywhere on the website? I don't think so

Reply:
I think they actually do show up on the website, just with the jurisdiction where the name of the state would be! 
<img width="370" alt="Screen Shot 2021-06-07 at 12 46 31 PM" src="https://user-images.githubusercontent.com/47676353/121065447-647b0780-c78e-11eb-87fc-9b598a871ce3.png">

Reply:
just checking, since these are aggregated they wouldn't show up here right? this is only for "facilities"

Reply:
that's my understanding!